hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -31,7 +31,7 @@
                // otherwise. The double-escaped quotes are because this needs to go inside a string
                // that gets passed to Make (stripping one level of quotes), and then needs to
                // result in an argument with an embedded space (the other one).
-               "-CFLAGS \\\"--std=c99 -fno-extended-identifiers\\\"",
+               "-CFLAGS \\\"--std=c++11 -fno-extended-identifiers\\\"",
```

Comment:
What about Xcelium?

Reply:
haven't used xcelium in top-level for a while. probably need to make some separate fixes for it

Reply:
I was wondering if we should proactively add the same switch to Xcelium as well, but its fine, not a priority.

Reply:
Xcelium option may be different from VCS. I created an issue #4230 for this.